backend/mydb.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const assert = require('assert');
+
+//import users
+const users = require('./user');
+ 
+//import dates
+const dates = require('./date');
+
+//connect
+mongoose.connect('mongodb://localhost:27017/mydb',
```

Comment:
I recommend extracting this hardcoded string to a constant. For example:
`MONGODB_URL = 'mongodb://localhost:27017/mydb'`